docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,462 @@
+| [`router.refresh`](#routerrefresh)                                  | Revalidate   |                       |                       |             |
+| [`fetch`](#fetch)                                                   |              |                       | Cache                 | Cache       |
+| [`fetch.options.cache`](#fetchoptionscache)                         |              |                       | Cache or Opt out      |             |
+| [`fetch.options.next.revalidate`](#fetchoptionsnextrevalidate)      |              | Revalidate            | Revalidate            |             |
+| [`fetch.options.next.tags`](#fetchoptionsnexttag-and-revalidatetag) |              | Cache                 | Cache                 |             |
+| [`revalidateTag`](#fetchoptionsnexttag-and-revalidatetag)           |              | Revalidate            | Revalidate            |             |
+| [`revalidatePath`](#revalidatepath)                                 |              | Revalidate            | Revalidate            |             |
+| [`const revalidate`](#segment-config-options)                       |              | Revalidate or Opt out | Revalidate or Opt out |             |
+| [`const dynamic`](#segment-config-options)                          |              | Cache or Opt out      | Cache or Opt out      |             |
+| [`cookies`](#cookies)                                               | Revalidate   | Opt out               |                       |             |
+| [`headers`, `useSearchParams`, `searchParams`](#dynamic-functions)  |              | Opt out               |                       |             |
+| [`React.cache`](#react-cache-function)                              |              |                       |                       | Cache       |
+| [`unstable_cache`](#unstable_cache) (Coming Soon)                   |              |                       |                       |             |
+
+### `<Link>`
+
+By default, the `<Link>` component automatically prefetches static routes (cached) and adds the React Payload to the Router Cache.
+
+To disable prefetching, you can set the `prefetch` prop to `false`. But this will not skip the cache permanently, the route segment will still be cached client-side when the user visits the route.
+
+Learn more about the [`<Link>` component](/docs/app/api-reference/components/link).
+
+### `router.prefetch`
+
+The `prefetch` option of the `useRouter` hook can be used to manually prefetch a route. This adds the React Payload to the Router cache.
+
+See the [`useRouter` hook](/docs/app/api-reference/functions/use-router) API reference.
+
+### `router.refresh`
+
+The `refresh` option of the `useRouter` hook can be used to manually refresh a route. This completely clears the Router Cache, and makes a new request to the server. `refresh` does not revalidate the Data or Render Cache, it reuses the cached data and rendering result, if it exists.
```

Comment:
```suggestion
The `refresh` function of the `useRouter` hook can be used to manually refresh a route. This completely clears the Router Cache, and makes a new request to the server for the current page you're on. `refresh` does not revalidate the Data or Full Route Cache, it reuses the cached data or React Server Component Payload if it exists.
```

Reply:
Good point, on option / function. Let's change it to method. I think that's nicer.